src/main/app/case/case-api.test.ts

```diff
@@ -316,6 +333,11 @@ describe('CaseApi', () => {
     const isApplicant2 = await api.isApplicant2('1234123412341234', userDetails.id);
     expect(isApplicant2).toBe(true);
   });
+
+  test('preSubmittedStatePrioritySequence should be appropriate', async () => {
```

Comment:
I think you can test this through the result of Submitted not being in the preSubmittedStatePrioritySequence, rather than exporting it just for testing